modules/ROOT/pages/netsuite/netsuite-reference.adoc

```diff
@@ -47,21 +47,27 @@ Release Notes: xref:release-notes::connector/netsuite-connector-release-notes-mu
 [%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Read Timeout a| Number |  Specifies the amount of time in milliseconds that the client waits for a response before it times out. 0 specifies that the client waits indefinitely. |  10000 | 
-| Connection Timeout a| Number |  Specifies the amount of time in milliseconds that the client attempts to establish a connection before it times out. 0 specifies that the client continues to attempt to open a connection indefinitely. |  10000 | 
-| Web Service Endpoint a| String |  Service endpoint |  | 
-| TLS configuration a| <<Tls>> |  Protocol to use for communication. Valid values are HTTP and HTTPS. Default value is HTTP. When using HTTPS the HTTP communication is secured using TLS / SSL. If HTTPS was configured as protocol then the user needs to configure at least the keystore in the tls:context child element of this listener-config. |  | 
-| Host a| String |  The hostname of the HTTP proxy, for example localhost This field also requires the port to be set. |  | 
-| Port a| Number |  The port number of the HTTP proxy, for example 3128. The port number must be specified if the hostname is also specified. |  | 
-| Username a| String |  The username which should be supplied to the HTTP proxy on every request to NetSuite. This field is optional, since a user might want to pass through an unauthenticated HTTP proxy. |  | 
-| Password a| String |  The password which would be supplied to the HTTP proxy on every request to NetSuite. This field is optional, since a user might want to pass through an unauthenticated HTTP proxy. |  | 
+| Read Timeout a| Number |  Specifies the amount of time in milliseconds that the client waits for a response before it times out. 0 specifies that the client waits indefinitely. |  10000 |
+| Connection Timeout a| Number |  Specifies the amount of time in milliseconds that the client attempts to establish a connection before it times out. 0 specifies that the client continues to attempt to open a connection indefinitely. |  10000 |
+| Web Service Endpoint a| String a|  The endpoint for the NetSuite web service. The hostname is specific to the user’s Account ID. The rest of the URL contains the API version of the site. For example:
+{sp} +
+{sp} +
+`http://user.netsuite.com/tstNetSuite_2019`
+{sp} +
+{sp} +
+If the call to the specified endpoint does not complete, the connector uses the remote URL, which it obtains automatically. |  |
+| TLS configuration a| <<Tls>> |  Protocol to use for communication. Valid values are HTTP and HTTPS. Default value is HTTP. When using HTTPS the HTTP communication is secured using TLS / SSL. If HTTPS was configured as protocol then the user needs to configure at least the keystore in the tls:context child element of this listener-config. |  |
+| Host a| String |  The hostname of the HTTP proxy, for example localhost This field also requires the port to be set. |  |
```

Comment:
for example localhost This field > for example localhost**.** This field
Missing (.dot)